<launch>
    <rosparam file="$(find mpc_casadi)/params/constraints.yaml" command="load" />

    <!-- Trajectory -->
    <node pkg="mpc_casadi" type="trajectory.py" name="trajectory" output="screen"/>

    <!-- Controller -->
    <node pkg="mpc_casadi" type="nmpc_node.py" name="nmpc_node" output="screen"/>

</launch>
